### PR TITLE
Adds paper label visuals to closets and lockers

### PR DESCRIPTION
**Pull request by SpaceRox1244**
## About the PR

Adds visuals for attaching a paper, like a cargo bounty or invoice, to a closet.

## Why / Balance

Closets can accept paper labels the same way that crates can, but they dont currently have a visual for it. I want to be able to label my department locker so nobody touches it!!

## Media

labels

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl: SpaceRox1244
- add: Closets and lockers now have visuals for being labeled with papers.

### CODE REVIEW

**Comment by beck-thompson** [comment]
I feel like CL is necessary its a cool feature

**Review by SaphireLattice — APPROVED** [review]
Hah. Actually wanted to suggest this sort of thing as an alternative to the pen eject PR. Though my thought was to suggest adding more colored papers. Something for a future PR, I suppose.

Quite like it! I do wish there were more paper types and the visualizer was actually uh, not needing a sprite for each different paper... Probably not quite in scope of this, but does highlight that issue more.

Never considered that the lockers could just have the paper label slapped on them, that makes an idea of "lockers label themselves if you use an ID with action (not ID locked!)" I've had even less relevant.

A personal question: any chance morgue could be added? Not required, just a "nice-to-have".

**Comment by SpaceRox1244** [comment]
> A personal question: any chance morgue could be added? Not required, just a "nice-to-have".

This PR was actually originally going to be about being able to label a whole bunch of new stuff, but I left it at the locker for now because I wanted to work on something else. I will probably come back to it soon.

**Comment by RedBookcase** [comment]
Sick. It always felt sort of unintended that you were able to label lockers, glad to see this added.

File: Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/base_structurelockers.yml [hunk 1]
```diff
@@ -20,6 +20,26 @@
...
+          False: { visible: false }
+      enum.StorageVisuals.Open:
+        enum.PaperLabelVisuals.Layer:
+          True: { visible: false }
+      enum.PaperLabelVisuals.LabelType:
+        enum.PaperLabelVisuals.Layer:
+          Paper: { state: paper }
+          Bounty: { state: bounty }
+          CaptainsPaper: { state: captains_paper }
+          Invoice: { state: invoice }
```
Comment (chromiumboy) [inline Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/base_structurelockers.yml:42]:
```suggestion

```
Lockers inherit this component and its visuals from closets, so no need to duplicate it

**Review by chromiumboy — CHANGES_REQUESTED** [review]
Just one minor thing from me

File: Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/base_structurelockers.yml (hunk 1 above)
Comment (SpaceRox1244) [reply Resources/Prototypes/Entities/Structures/Storage/Closets/Lockers/base_structurelockers.yml]: Thanks for the catch! I took care of it.

**Comment by TiniestShark** [comment]
Honestly glad to have this with the amount of times I've grabbed lockers on Oasis for bounties only for the rest of cargo to not know that was a thing

**Comment by chromiumboy** [comment]
Awesome, its good to go

**Review by chromiumboy — APPROVED** [review]

**Comment by Nettingman** [comment]
This is great! I love how many times it happens, that I feel like a feature would be cool in Space Station 14 and turns out someone already made a pullrequest for it.

**Pull request merged**